contracts/ShortNameClaims.sol

```diff
@@ -52,30 +54,60 @@ contract ShortNameClaims is Ownable {
         Status status;
     }
 
+    Roles.Role owners;
+    Roles.Role ratifiers;
+
     PriceOracle public priceOracle;
     BaseRegistrar public registrar;
     mapping(bytes32=>Claim) public claims;
     mapping(bytes32=>bool) approvedNames;
     uint public pendingClaims;
     uint public unresolvedClaims;
     Phase public phase;
-    address public ratifier;
 
     event ClaimSubmitted(string claimed, bytes dnsname, uint paid, address claimant, string email);
     event ClaimStatusChanged(bytes32 indexed claimId, Status status);
 
     constructor(PriceOracle _priceOracle, BaseRegistrar _registrar, address _ratifier) public {
         priceOracle = _priceOracle;
         registrar = _registrar;
-        ratifier = _ratifier;
         phase = Phase.OPEN;
+
+        owners.add(msg.sender);
+        ratifiers.add(_ratifier);
```

Comment:
What happens if ratifier is empty?

Reply:
Then there are no ratifiers, and we can't proceed to the next stage.